### PR TITLE
Add libnvidia-rtcore as Nvidia library

### PR DESCRIPTION
**Pull request by joedborg**
Following on from https://github.com/snapcore/snapd/pull/8397.  There is a need for this library too.

### CODE REVIEW

**Comment by joedborg** [comment]
cc @anonymouse64 @jdstrand

File: cmd/snap-confine/mount-support-nvidia.c [hunk 1]
```diff
@@ -113,6 +113,7 @@ static const char *nvidia_globs[] = {
 	"libnvidia-ml.so*",
 	"libnvidia-opencl.so*",
 	"libnvidia-ptxjitcompiler.so*",
+	"libnvidia-rtcore.so*",
```
Comment (jdstrand) [inline cmd/snap-confine/mount-support-nvidia.c:116]: Like in https://github.com/snapcore/snapd/pull/8397/files, I believe we also need a corresponding change in opengl.go.

**Comment by joedborg** [comment]
Thanks @mvo5.   Whilst this makes its way to edge, is there a way I can manually copy this somewhere that'll get picked up?  Seeing as https://github.com/snapcore/snapd/blob/df6919d946f42c48f2e388c7446e626178a894a5/interfaces/builtin/opengl.go#L40 will still pick this up.

File: cmd/snap-confine/mount-support-nvidia.c (hunk 1 above)
Comment (joedborg) [reply cmd/snap-confine/mount-support-nvidia.c:116]: https://github.com/snapcore/snapd/blob/df6919d946f42c48f2e388c7446e626178a894a5/interfaces/builtin/opengl.go#L40 Will this wildcard not pick it up?

Comment (jdstrand) [reply cmd/snap-confine/mount-support-nvidia.c:116]: Oh, https://github.com/snapcore/snapd/pull/8419#issuecomment-608503207 pointed out that it is already picked up by this rule:
```
/var/lib/snapd/hostfs/{,usr/}lib{,32,64,x32}/{,@{multiarch}/}libnvidia*.so{,.*} rm,
```

Approving.

**Comment by mvo5** [comment]
@joedborg It's a little bit tricky to test this manually, if you use the snapd snap it should be available in ~30min on edge. If you are not using the snapd snap just "sudo snap install snapd" and snapd will use that for it's tooling.

**Pull request merged**